indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -501,9 +501,38 @@ th.i-table {
         }
     }
 
-    .table-content {
+    thead {
+        display: block;
+        position: relative;
+        background-color: $light-gray;
+        border-bottom: 1px solid $gray;
```

Comment:
Simply use `@include border-bottom();` like we do everywhere else with `border-all()`. It will apply by default `border-bottom: 1px solid $gray` and won't need any manual update if we ever change how borders should look Indico-wide.